scripts/linux/trcE2ETest.sh

```diff
@@ -917,7 +919,7 @@ function configure_connectivity_settings() {
 
     TEST_INFO="$TEST_INFO,TestDuration=${TEST_DURATION}"
 
-    log_upload_enabled=false
+    log_upload_enabled=true
```

Comment:
We don't need to upload log for most cases. Only when you're testing the upload (or if the test result fails.)